File: grok/benchmarks/fib.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

def main():
    start = time.time()
    result = fib(30)
    end = time.time()
    print(f"Result: {result}")
    print(f"Time: {end - start:.4f}s")

if __name__ == "__main__":
    main()

File: grok/benchmarks/fib.rs
use std::time::Instant;

fn fib(n: i32) -> i32 {
    if n < 2 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let start = Instant::now();
    let result = fib(30);
    let duration = start.elapsed();
    println!("Result: {}", result);
    println!("Time: {:.4}s", duration.as_secs_f64());
}
